Resources/Prototypes/Entities/Mobs/Species/harpy.yml

```diff
@@ -119,8 +119,8 @@
           False:  {visible: false}
           True:  {visible: true}
   - type: MovementSpeedModifier
-    baseWalkSpeed: 2.5
-    baseSprintSpeed: 5.0
+    baseWalkSpeed: 3
+    baseSprintSpeed: 5.5
```

Comment:
Ditto about changing speed

Reply:
Glad to see we actually kept that "Harpies move 10% faster than default human", and was accounted for by this PR.